WalletWasabi/Extensions/RPCClientExtensions.cs

```diff
@@ -63,8 +63,8 @@ private static FeeRateByConfirmationTarget SimulateRegTestFeeEstimation() =>
 	/// </summary>
 	public static async Task<AllFeeEstimate> EstimateAllFeeAsync(this IRPCClient rpc, CancellationToken cancel = default)
 	{
-		var smartEstimations = rpc.Network == Network.RegTest
-			? SimulateRegTestFeeEstimation()
+		var smartEstimations = rpc.Network != Network.Main
```

Comment:
I don't understand this change. This was always working before, why is this necessary for testnet4?